examples/demo-stf/README.md

```diff
@@ -0,0 +1,99 @@
+## Enabling RPC via SDK Macros
+There are 5 steps that need to be completed to enable RPC on the full node
+1. Annotate the modules that need to expose their data with `rpc_gen` and `rpc_method`
+2. Annotate the state transition runner with the specific modules to expose with `expose_rpc`
+3. Implement the `RpcRunner` trait. provide an implementation for the `get_storage` function
+4. Import and call `get_rpc_module` to get a combined rpc module for the modules annotated and exposed in 1 and 2
+5. Use the modules returned from the above function and bind them to an RPC server
+
+### Modules
+* We need to annotate the `impl` block for our module. In this case its `ModuleStruct`
+```
+impl<C: Context> MyModule<C> {
```

Comment:
Nit: Instead of `MyModule` can we use a real module used in `demo-stf`? Below I see you use `DemoAppRunner` which is a real code we have.

Reply:
sure, i can actually replace MyModule with Bank